lightning/src/chain/channelmonitor.rs

```diff
@@ -3987,9 +4009,7 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		}
 	}
 
-	/// Check if any transaction broadcasted is paying fund back to some address we can assume to own
-	fn is_paying_spendable_output<L: Deref>(&mut self, tx: &Transaction, height: u32, block_hash: &BlockHash, logger: &L) where L::Target: Logger {
-		let mut spendable_output = None;
+	fn get_spendable_output(&self, tx: &Transaction) -> Option<SpendableOutputDescriptor> {
 		for (i, outp) in tx.output.iter().enumerate() { // There is max one spendable output for any channel tx, including ones generated by us
```

Comment:
Just checking: this comment is still true now that users can pass in arbitrary txs, right?

Reply:
Makes sense to me. IMO `ChannelMonitorImpl::counterparty_payment_script`, etc could use doc comments but that's pre-existing.